src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
As properties are not the recommended way to configure this plugin, I am choosing the second option, which makes the target for the property a bit more clear. If you are moving them from properties to the plugin-configuration, you would normally tend to "just remove the `jfx.`-prefix, which will end in a wrong result ... thats why I prefer the second option